dgraphtest/cluster.go

```diff
@@ -297,7 +297,9 @@ loop:
 		time.Sleep(waitDurBeforeRetry)
 
 		resp, err := c.AlphasHealth()
-		if err != nil {
+		if err != nil && strings.Contains(err.Error(), "the server is in draining mode") {
```

Comment:
Can you make the error string a constant. Both here, and where it is emitted.

Reply:
The error string is actually much longer. I don't know if we want to compare such a long string here.